CLASSES:
  - car
  - bus
  - truck
  - pedestrian
  - bimo

RANGE:
  X_MIN: 0
  X_MAX: 89.6
  Y_MIN: -11.2
  Y_MAX: 11.2
  Z_MIN: -3
  Z_MAX: 3

VOXEL_SIZE:
  - 0.2
  - 0.2
  - 0.2

BATCH_SIZE: 1

MODEL_PATH: model/livoxmodel

OVERLAP: 11.2

GPU_INDEX: 0
NMS_THRESHOLD: 0.1
BOX_THRESHOLD: 0.6

tower_confg:
  alpha: 0
  theta: 12
  gamma: 0
  x: 0
  y: 0
  z: 7